chaertli/migrations/0003_sitae.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-04-20 17:35
+from __future__ import unicode_literals
```

Comment:
[Future statements](https://docs.python.org/3/reference/simple_stmts.html#future-statements)

Further reading: http://stackoverflow.com/questions/7075082/what-is-future-in-python-used-for-and-how-when-to-use-it-and-how-it-works